lagoon-remote/harbor-clair

```diff
@@ -0,0 +1 @@
+/Users/cdchris12/amazeeprojects/lagoon/services/harbor-clair
```

Comment:
this should be a relative link, not an absolute one.

Reply:
Resolved in the latest commit.